bootstrap.php

```diff
@@ -2,8 +2,17 @@
 
 use Broadway\CommandHandling\CommandBus;
 use Broadway\EventHandling\EventBus;
+use CultuurNet\UDB3\AggregateType;
```

Comment:
Damn, this might cause some issue. PhpStorm orders them alphabetically during a move apparently.

Reply:
Should be fixed.